app/src/main/java/no/usn/mob3000/ui/components/base/TopNavBar.kt

```diff
@@ -50,10 +50,8 @@ fun TopNavbar(
     topBarActions: @Composable (RowScope.() -> Unit) = {},
 ) {
     TopAppBar(
-        title = { if (showTitle) Text(stringResource(currentScreen.title)) },
-        colors = TopAppBarDefaults.mediumTopAppBarColors(
-            containerColor = NavbarBackground
-        ),
+        title = { if (showTitle) Text(stringResource(currentScreen.title),color = MaterialTheme.colorScheme.onSurface) },
+        colors = TopAppBarDefaults.mediumTopAppBarColors(containerColor = MaterialTheme.colorScheme.surface),
```

Comment:
This should be indented. E.g.
```kotlin
TopAppBar(
        title = {
                if (showTitle) {
                        Text(stringResource(currentScreen.title),
                        color = MaterialTheme.colorScheme.onSurface)
                }
        },
        colors = TopAppBarDefaults.mediumTopAppBarColors(containerColor = MaterialTheme.colorScheme.surface),
```

(Note, tabs may appear larger on GitHub than they are supposed to be in-file).